gcp/modules/couchdb/main.tf

```diff
@@ -4,27 +4,41 @@ terraform {
 
 variable "env" {}
 variable "secrets_dir" {}
-variable "values_dir" {}
 variable "charts_dir" {}
 variable "nonce" {}
 
 # Terragrunt variables
 variable "couchdb_replicas" {}
+
 variable "backup_deltas" {}
 variable "release_namespace" {}
 
 # Secret variables
 variable "secret_couchdb_admin_username" {}
+
```

Comment:
Looks like an extra blank line snuck in here.

Reply:
Yeah, `terraform fmt` somehow thinks that it should be here.